### PR TITLE
Readme rework

### PR DESCRIPTION
**Pull request by creativecreatorormaybenot**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,98 +1,169 @@
...
+ * `ContentOrientation.below`: Text is displayed below the target.
+  
+ * `ContentOrientation.above`: Text is displayed above the target.
+
+#### `onTargetTap`
+
+```dart
+   onTargetTap: () async {
+    // Executed when the tap target is tapped. The overlay will not close before
+    // this function returns and after that, the next step will be opened.
```
Comment (axel-op) [inline README.md:66]: That's actually very useful to know!

File: README.md [hunk 2]
```diff
@@ -1,98 +1,169 @@
...
+    ); 
+  });
+  super.initState();
+}
 ```
-GlobalKey<EnsureVisibleState> ensureKey2 = GlobalKey<EnsureVisibleState>();
+
+#### Other methods
+
+You can view the [API reference for `FeatureDiscovery`](https://pub.dev/documentation/feature_discovery/latest/feature_discovery/FeatureDiscovery-class.html#static-methods) to find other useful methods for controlling the Feature Discovery process programmatically.
```
Comment (axel-op) [inline README.md:137]: Unfortunately documentation doesn't appear for static methods :(

**Review by axel-op — APPROVED** [review]
Love it.

File: README.md (hunk 2 above)
Comment (creativecreatorormaybenot) [reply README.md:137]: Well, there is none in the current version of the plugin (on Pub), but you are saying that it will never show?

Comment (axel-op) [reply README.md:137]: There is one!

Comment (creativecreatorormaybenot) [reply README.md:137]: In your version, but not [here](https://github.com/ayalma/feature_discovery/blob/master/lib/src/feature_discovery.dart#L15), which is what I linked to.

Comment (axel-op) [reply README.md:137]: Ooooh… Well, merge it now! Hoping @ayalma the original author of the plugin will do the same…

Comment (creativecreatorormaybenot) [reply README.md:137]: For my [`wakelock` plugin](https://pub.dev/documentation/wakelock/latest/wakelock/Wakelock-class.html#static-methods), there is documentation for the static methods.

Comment (creativecreatorormaybenot) [reply README.md:137]: >Hoping @ayalma the original author of the plugin will do the same… 

We will see. Their status says "I may be slow to respond", so we can wait I guess.

**Pull request merged**